manifests/t/TeamRadarr/Radarr/4.5.2.7388/TeamRadarr.Radarr.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 4.5.2.7388
+MinimumOSVersion: 10.0.0.0
+AppsAndFeaturesEntries:
+- DisplayName: Radarr

```

```suggestion
- DisplayName: Radarr version 4.5.2
```
MUST match control panel